developer-docs-site/docs/tutorials/full-node/troubleshooting-fullnode.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Troubleshooting FullNode Setup"
+slug: "troubleshooting-fullnode-setup"
+sidebar_position: 13
+---
+
+# Troubleshooting FullNode Setup
+
+**Q: When starting the node, it throws a YAML-parsing error.**
+
+**A:** YAML files are sensitive to formatting errors. Use a dedicated YAML editor or use a YAML syntax checker in your preferred editor to check if each line in the YAML file is indented correctly.
+
+**Q: When I start the node with `cargo run -p ...` command I get "Unable to fetch any peers to poll" error.**
+
+It looks like I have no peers on the available node testers. I have no output when I run:
+
+```
+curl 127.0.0.1:9101/metrics 2> /dev/null | grep "aptos_state_sync_version{type=\"synced\"}"
+```
+
+On the available node testers, it shows that I am not syncing.
```

Comment:
What's a node tester?

I'd say I have `Unable to fetch any peers` as well as my sync version doesn't increase.  Which means I'm not connected to the blockchain.

Possible reasons are bad network configuration (firewalls & security groups) or the connections are saturated on the validator full nodes.

1. Check your configuration
2. Add new seed peers